### ISSUE TITLE
S2S_I simultaneous work possible_ II. no change of jurisdiction_ III: strange error message

### ISSUE THREAD

**Issue by SahaLinaPrueger** [hzi-braunschweig/SORMAS-Project#9836]
### Bug Description

- [ ] I. If you follow the steps described above simultaneous work of two health departments on one case is possible. This works if you add an external ID AND/OR external token and internal token and general comment to the case. I did not tested if this is also possible it you only add an internal token or external token or general comment. Please figure this out. It is certain that without fill in this fields everything works correctly.
- [ ] II. If you follow the steps described above the jurisdiction does not change if you sent a case back from SORMAS B to SORMAS A.
- [ ] III. If you follow the steps described above you can not accept a case and a strange error message occurs. Maybe we have to reopen the ticket https://github.com/hzi-braunschweig/SORMAS-Project/issues/9078 ?
- [ ] IV: I guess fail of https://github.com/hzi-braunschweig/SORMAS-Project/issues/6875 and https://github.com/hzi-braunschweig/SORMAS-Project/issues/7014
### Steps to Reproduce

1. SORMAS A: Create a case on SORMAS A and add an external ID, external token, internal token and general comment + external ID of pathogen result (13.July.2022 13:56 release-de)
2. SORMAS A: Send the case to SORMAS B with hand over the ownership (13.July.2022 13:56 from release-de to release-sormas-x)
3. SORMAS B: Accept the case on SORMAS B (13.07.2022 13:56 release-sormas-x)
4. SORMAS B: Open the case on SORMAS B and check if external ID, external token, internal token and general comment + external ID of pathogen result are visible 
5. SORMAS B: Change the external token, internal token and general comment on SORMAS B 
6. SORMAS B: Save the case (13:58 release-sormas-x)
7. SORMAS A: Open the case on SORMAS A and check if fields are greyed out because SORMAS A has no longer the hand over the ownership and if the sync is correct.
8. SORMAS B: Open the case in SORMAS B and check the responsible jurisdiction.
9. SORMAS B: Open the case on SORMAS B and send the case back to SORMAS A with hand over the ownership (13.07.2022 14:01 release-sormas-x zu release-de)
10. SORMAS A: Accept the case on SORMAS A (13.07.2022 14:01 500 Internal Server Error)
11. SORMAS A: Reload page (result: accept button and reject button still there)
12. SORMAS B: Go back to SORMAS B and click on "save". (13.07.2022 14:02)
13. SORMAS A: Click on "accept" on SORMAS A (13.07.2022 14:02-14:05) 
14. SORMAS A: Open the case on SORMAS A and check if external ID, external token, internal token and general comment + external ID of pathogen result are visible (result: they are visible, + external ID of the case is editable)
15. SORMAS A: Open the case and check if responsible jurisdiction had changed (result: did not)
16. SORMAS B: Open the case on SORMAS B and check if fields are greyed out because SORMAS B has no longer the hand over the ownership (result: fields are not greyed out)
17. Try to edit data in SORMAS A and SORMAS B and see if they sync correctly (result: they sync correctly) 

### Expected Behavior

~Step 4: i guess external ID of pathogen result should not be visible according to https://github.com/hzi-braunschweig/SORMAS-Project/issues/7014 ~
Step 10: Error Message should not be there
Step 11: At least after reloading the page the error message should not be there. Maybe we have to reopen the ticket https://github.com/hzi-braunschweig/SORMAS-Project/issues/9078 ? But I am not sure if this is the same because the issue #9078  has been fixed with the error message "The request is already processed. Please reload the page to see the latest changes." (Error Message now is "Please reload the page to see the latest changes."
Step 14: I guess external ID, external token, internal token and general comment + external ID of pathogen result should not be visible according to https://github.com/hzi-braunschweig/SORMAS-Project/issues/7014 and https://github.com/hzi-braunschweig/SORMAS-Project/issues/6875 
Step 15: responsible jurisdiction has to change
Step 16: Case should not be editable on SORMAS B anymore. Fields have to be greyed out. Seems like the hand over the ownership still belongs to SORMAS B, that would explain why the jurisdiction does not change.
### Screenshots

18.07.2022 14:01 500 Internal Server Error
![image](https://user-images.githubusercontent.com/100844689/178729025-3fe09ac8-8986-49e9-aec9-eea7494ae460.png)

### System Details

* Device: Windows 11
* SORMAS version: 1.73.2
* Android version/Browser: Edge
* Server URL: release-de and release-sormas-x
* User Role: admin

### Additional Information

I combined the four bugs in one Issue because maybe they are related to each other. So you have everything together in one ticket to be able to understand the connections better.

The following properties are not set on release-de and release-sormas-x:
#sormas2sormas.ignoreProperty.additionalDetails=true
#sormas2sormas.ignoreProperty.externalId=true
#sormas2sormas.ignoreProperty.externalToken=true
#sormas2sormas.ignoreProperty.internalToken=true
As described in comments of Issue #6875:  _By uncommenting them and setting them to false it's the only way to allow the system to transmit/receive them._ 

I wanted to test this for contacts, too. But: https://github.com/hzi-braunschweig/SORMAS-Project/issues/9784 
Please feel free to check this bug for contacts, too. I have no access to Version 1.74 with S2S right now. (not until 1.74 is on our RC Candidates)

**Comment by SahaLinaPrueger** [comment]
@JaquM-HZI

**Comment by SahaLinaPrueger** [comment]
Important correction: 
The following properties are not set on release-de and release-sormas-x:
#sormas2sormas.ignoreProperty.additionalDetails=false
#sormas2sormas.ignoreProperty.externalId=false
#sormas2sormas.ignoreProperty.externalToken=false
#sormas2sormas.ignoreProperty.internalToken=false

so Point IV. fail of #6875 #7014 is invalid and can be ignored.

**Comment by JonasCir** [comment]
We cannot proceed with this ticket in its current form. There is to much going on here and we need to split this into smaller chunks. From the logs I received, I could identify that the 500 Internal Server Error originates from the converter. I do not have a SurvNet Instance available as I'm running Linux so I cannot create a minimal reproduction for this. I assume, however, that sharing back a case with the exact same instance is to blame here. From looking at the code, I'm quite certain that the accept/reject feature can be disabled which should make the reproduction shorter.

Cannot say anything about the other issues mentioned, need to investigate further.

**Comment by JonasCir** [comment]
Refreshing the page won't help here as the error is reported back from the remote. So I think the only thing we can do is to improve the error message and make clear to the user that the error is happening on the remote, not locally on its instance.

**Comment by JonasCir** [comment]
Test pass for me locally closing
